### PR TITLE
Upgrading to CircleCI 2.0

### LINKED ISSUE #385: Migrate to circle-ci 2

### PR DESCRIPTION
**Pull request by jrgriffiniii**
Resolves #385

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      - image: postgres:10.3
+        environment:
+          POSTGRES_DB: dpul_test
+          POSTGRES_USER: pomegranate
+          POSTGRES_PASSWORD: ""
+    steps:
+      - attach_workspace:
+          at: '~/pomegranate'
+      - run: sudo apt-get update
+      - run: sudo apt-get install libpq-dev libopenjp2-7-dev libmagickwand-dev imagemagick pkg-config
```
Comment (escowles) [inline .circleci/config.yml]: I don't see any of these being installed in the v1 config, and they aren't mentioned in the README.  Do we want to document additional dependencies that surfaced here?

File: config/database.yml [hunk 2]
```diff
@@ -15,8 +15,10 @@ development: &default
 
 test:
   <<: *default
-  database: dpul_test
-
+  database: <%= ENV["POMEGRANATE_DB"] %>
+  host: <%= ENV["POMEGRANATE_DB_HOST"] %>
+  username: <%= ENV["POMEGRANATE_DB_USERNAME"] %>
+  password: <%= ENV["POMEGRANATE_DB_PASSWORD"] %>
```
Comment (escowles) [inline config/database.yml]: Are there any changes needed to run the tests locally?  Could we fallback on the old defaults?

File: .circleci/config.yml (hunk 1 above)
Comment (jrgriffiniii) [reply .circleci/config.yml]: These were installed based upon my findings within other projects.  I had falsely assumed them to be necessary.

File: config/database.yml (hunk 2 above)
Comment (jrgriffiniii) [reply config/database.yml]: My apologies, there were errors in this configuration; I have corrected these, and can now run the tests locally without error.

**Pull request merged**